WordPress/src/main/res/layout/prepublishing_toolbar.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
So while I was working on the adding a toolbar to the `Tags` screen I realized that the default toolbar was giving transparency issues and such forth so I thought it would make sense to just create the layout. WIP version. Let me know if this approach is fine. Especially from a code reusability standpoint. Based on the problem set only two screens will use the toolbar as the main screen will use a customized header so I wasn't sure if I should build a whole component just to do that.

Reply:
Since it's technically not an Android toolbar I think this is a good enough solution (at least for now). I just noticed the title is not horizontally centered.

Reply:
Thanks. I will work on making the necessary changes to complete it. Yes, the title alignment needs work. Will finalize it 🙏